### ISSUE TITLE
With multiple desktop spaces, script does not activate the correct Chrome window

### ISSUE THREAD

**Issue by voidl** [mobjack/MacChromeCalendarFocus#1]
I have two desktop spaces with Chrome windows present on both spaces. If I am on space 1, and Chrome window with desired tab exists on space 2, then running the script will not activate the window on space 2.
Have you had that issue?

**Comment by voidl** [comment]
At second glance, the script seems to fail even if multiple chrome windows are open on the same desktop space..

**Comment by mobjack** [comment]
True, I only keep one calendar window open.  The script will find the first one.  Feel free to branch it and see if you can find the one on window 1.  Thanks.
